From 418ae93263d882a5612e3ff922accd7af7629e41 Mon Sep 17 00:00:00 2001
From: Cynthia Huang <cynthia@andestech.com>
Date: Mon, 28 Oct 2024 09:11:47 +0800
Subject: [PATCH 130/168] riscv: andes: remove useless config option (#217)

Remove input-related config options in ltp.config.
Eevn with the input turned on, we still do not have an input device to perform LTP test.

Reviewed-on: https://gitea.andestech.com/RD-SW/linux/pulls/217
Reviewed-by: Charles Ci-Jyun Wu <dminus@andestech.com>
Co-authored-by: Cynthia Huang <cynthia@andestech.com>
Co-committed-by: Cynthia Huang <cynthia@andestech.com>
---
 arch/riscv/configs/ltp.config | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/riscv/configs/ltp.config b/arch/riscv/configs/ltp.config
index f0cb3a47ea0d..5b157b1f0598 100644
--- a/arch/riscv/configs/ltp.config
+++ b/arch/riscv/configs/ltp.config
@@ -25,8 +25,6 @@ CONFIG_SWAP=y
 CONFIG_CAN=y
 CONFIG_CAN_DEV=m
 CONFIG_CAN_VCAN=m
-CONFIG_INPUT_MISC=y
-CONFIG_INPUT_UINPUT=m
 CONFIG_USER_DECRYPTED_DATA=y
 CONFIG_ROMFS_BACKED_BY_BLOCK=y
 CONFIG_MISC_FILESYSTEMS=y
-- 
2.34.1

